<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-09-03T19:03:37 IST" name="Suite" finished-at="2023-09-03T19:03:45 IST" duration-ms="7421">
    <groups>
    </groups>
    <test started-at="2023-09-03T19:03:37 IST" name="Test" finished-at="2023-09-03T19:03:45 IST" duration-ms="7421">
      <class name="com.Files.FileScreenshot">
        <test-method is-config="true" signature="setup()[pri:0, instance:com.Files.FileScreenshot@1f2586d6]" started-at="2023-09-03T19:03:37 IST" name="setup" finished-at="2023-09-03T19:03:43 IST" duration-ms="6012" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="verfiy()[pri:0, instance:com.Files.FileScreenshot@1f2586d6]" started-at="2023-09-03T19:03:43 IST" name="verfiy" finished-at="2023-09-03T19:03:44 IST" duration-ms="322" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Hello (Adele song) - Wikipediaa] but found [Hello (Adele song) - Wikipedia]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Hello (Adele song) - Wikipediaa] but found [Hello (Adele song) - Wikipedia]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at com.Files.FileScreenshot.verfiy(FileScreenshot.java:45)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verfiy -->
        <test-method is-config="true" signature="AfterMethod(org.testng.ITestResult)[pri:0, instance:com.Files.FileScreenshot@1f2586d6]" started-at="2023-09-03T19:03:44 IST" name="AfterMethod" finished-at="2023-09-03T19:03:44 IST" duration-ms="682" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verfiy status=FAILURE method=FileScreenshot.verfiy()[pri:0, instance:com.Files.FileScreenshot@1f2586d6] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- AfterMethod -->
        <test-method is-config="true" signature="TearDown()[pri:0, instance:com.Files.FileScreenshot@1f2586d6]" started-at="2023-09-03T19:03:44 IST" name="TearDown" finished-at="2023-09-03T19:03:45 IST" duration-ms="307" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TearDown -->
      </class> <!-- com.Files.FileScreenshot -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
